### PR TITLE
Add documentation for access log plugin

### PR DESCRIPTION
**Pull request by gargnupur**
**What this PR does / why we need it**:
Add documentation for access log plugin

### CODE REVIEW

File: extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto [hunk 1]
```diff
@@ -15,10 +15,19 @@
 
 syntax = "proto3";
 
+// $title: AccessLogPolicy Config
+// $description: Configuration for AccessLogPolicy Filter.
+// $location:
+// https://istio.io/docs/reference/config/proxy_extensions/accesslogpolicy.html
+// $weight: 20
+// This filter logs all error requests. Success requests are repeatedly logged
```
Comment (mandarjog) [inline extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto:23]: This filter is used in conjunction with another filter.
Can you add more details here?
Which attribute is produced, and how another filter is expected to use it.

Comment (gargnupur) [reply extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto:23]: Done.

**Comment by gargnupur** [comment]
@mandarjog , @bianpengyuan : it's ready for review again..

**Comment by bianpengyuan** [comment]
Block in case @mandarjog want to take a look.

File: extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto (hunk 1 above)
Comment (mandarjog) [reply extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto:23]: Accesslog Policy plugin is a stateful http log sampler. It decides whether a request is logged based on the following rules.
1. All requests resulting in errors are logged.
2. First successful request within `log_window_duration ` from a specific source ip (source principal) is  logged.

The plugin records its decision in the `access_log_policy` with a value of  `"no"`. A downstream plugin may honor the  the attribute. For example, `Stackdriver` plugin will not produce an access log entry if this attribute is set.

Comment (gargnupur) [reply extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto]: Thanks! Changed.. this is much better :)

**Comment by mandarjog** [comment]
@gargnupur @kyessenov this Plug-in predates our “istio.” Convention for injecting new attributes. So for this has just been an internal contract between stack driver and accesslog policy.

Here is what I suggest 
1. Change the name of the attribute to “istio.accesslog_sampled” 
2. Update the name in the SD plugin
3. Update the proto doc to talons our “attribute”, at present it does not mention this word.

File: extensions/metadata_exchange/config.pb.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Metadata Exchange Config
 description: Configuration for Metadata Exchange Filter.
-location: https://istio.io/docs/reference/config/proxy_extensions/metadata_exchange.html
+location: https://istio.io/docs/reference/config/proxy_extensions/metadata_exchange.html.
```
Comment (richardwxn) [inline extensions/metadata_exchange/config.pb.html:4]: is this changed added manually or updated by make extensions-docs?

Comment (gargnupur) [reply extensions/metadata_exchange/config.pb.html:4]: updated using make extension-docs.. I can revert this change?

**Comment by gargnupur** [comment]
@mandarjog , @kyessenov : added "istio." in access log attribute ..
PTAL

**Pull request merged**